### DISCUSSION TITLE
Allow other keyboard navigation keys to work in menus

### DISCUSSION THREAD

**Discussion by advice2020** [doublecmd/doublecmd#1690] [Suggestions]
I have noticed that when in any menus certain navigation keys are not supported.
Menus such as.....
-Menu Bar dropdowns
-Context menus
-Button menus
-Etc.

Navigation keys not supported...
-Home / End
-Page Up / Page Down

DC does allow arrows when cursor is at top or bottom of menus to go to the top or bottom of the list.
This does help at times, but other times can be tricky if you hold arrow keys down.
Other navigational keys could be helpful.
If they are added, not sure if would keep the cyclic cursor action or not, overall I don't think they would conflict with each other.

Anyways, thank you to anyone who reads this

**Comment by alexx2000** [comment]
Menu is a system control. DC cannot change menu behavior.

**Comment by advice2020** [comment]
@alexx2000 
Thank you for your response Alex.
You do not have to answer this next question, if it cannot be done then it cannot be done, but what I do not understand is that if this is the case, then why do all of these navigation keys work in all these menu types in my DE.  Only thing I can think of is maybe because my DE is GTK based and the DC version I am using is QT based.
Thank You again for your response, like I said you do not have to answer this.

**Comment by alexx2000** [comment]
> Only thing I can think of is maybe because my DE is GTK based and the DC version I am using is QT based.

Yes, GTK menu has this feature, Qt not.

**Comment by advice2020** [comment]
@alexx2000 
Ahh, thank you Alex, I do not use that many Qt based applications so I did not realize that.
Kind of ironic that Qt is known for allowing a great deal of customization and does not have something like this :)
I was going to mention on their Github page, but only a page for Qt5 and there does not appear to be an "Issues" or "Discussions" button at the top so they must not be interested in user feedback on their Github itself.
Anyways, thank you again for your original response and for your response confirming my assumption.
